app/build.gradle

```diff
@@ -26,8 +26,8 @@ android {
 
         minSdkVersion 24
         targetSdkVersion 26
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/b59edc35dded2ffaa1e71b74a0eae44fd295c7a8/app/build.gradle#L28"></span>Not targeting the latest versions of Android; compatibility <br />
modes apply. Consider testing and updating this version. <br />
Consult the android.os.Build.VERSION_CODES javadoc for details.</td>
    </tr>
  </tbody>
</table>